src/views/PeopleSearch/index.js

```diff
@@ -118,7 +118,7 @@ const peopleSearchHeaderDesktop = (
         <Typography variant="body2" style={styles.headerStyle}>
           @GORDON.EDU
           <br />
-          MAIL LOCATION
+          MAILBOX #
```

Comment:
I'm not sure why this change was made, but it will be confusing for Fac/Staff results now because we have mailstops instead of mailboxes. For example, my mailstop in prod is CTS, so `Mailbox #CTS` doesn't make much sense.